src/main/java/domain/GameSystem.java

```diff
@@ -0,0 +1,53 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import utils.RandomUtils;
+import view.InputView;
+import view.OutputView;
+
+public class GameSystem {
+
+    final static private int START_RANDOM_NUM = 0;
+    final static private int END_RANDOM_NUM = 10;
```

Comment:
```suggestion
    private static final int START_RANDOM_NUM = 0;
    private static final int END_RANDOM_NUM = 10;
```
보통 접근제한자가 제일 앞에 있습니다.